src/core/namespace.c

```diff
@@ -2286,13 +2287,27 @@ int setup_tmp_dirs(const char *id, char **tmp_dir, char **var_tmp_dir) {
         return 0;
 }
 
-int setup_netns(const int netns_storage_socket[static 2]) {
-        _cleanup_close_ int netns = -1;
+static const char *ns_filename(unsigned long nstype) {
+        unsigned i;
+
+        for (i = 0; namespace_flag_map[i].name; i++)
```

Comment:
Please declare the variable in the loop: `for (unsigned i = 0; namespace_flag_map[i].name; i++)`